src/shapes/polyline.class.ts

```diff
@@ -185,16 +212,25 @@ import { degreesToRadians } from '../util/misc/radiansDegreesConversion';
        * @private
        */
       _getTransformedDimensions: function (options) {
-        return this.exactBoundingBox
-          ? this.callSuper('_getTransformedDimensions', {
-              ...(options || {}),
-              // disable stroke bbox calculations
-              strokeWidth: 0,
-              // disable skewing bbox calculations
-              skewX: 0,
-              skewY: 0,
-            })
-          : this.callSuper('_getTransformedDimensions', options);
+        if (this.exactBoundingBox) {
+          // calculate size
+          const { width, height } =
+            options && !(options.width && options.height)
```

Comment:
`_getTransformedDimensions` is used by the skewing control logic so it needs to calculate size properly
However passing width/height is not supported unless we apply scaling after projecting the points. What do you think?

Reply:
we should unit tests for `_getTransformedDimensions`

Reply:
It is actually silly
No one should pass width height to a function that calculates them
I got confused